pages/new.tsx

```diff
@@ -1,25 +1,22 @@
-import { Box, Text, Heading } from '@chakra-ui/react'
-import StoryForm from '../components/form'
 import Link from 'next/link'
+import { Text, Heading } from '@chakra-ui/react'
+import ContentBox from '../components/common/ContentBox'
+import StoryForm from '../components/stories/StoryForm'
 
 export default function CreateStory() {
   return (
-    <div>
-      <Box w="100%" p={8} pb={24} maxWidth="4xl">
-        <Box pb={4}>
-          <Heading as="h1" size="lg" pb={4}>
-            Share your COVID story
-          </Heading>
-          <Text>
-            Every number has a story. We believe in the power of storytelling to force government
-            action in Ontario. We value your privacy.
-          </Text>
-          <Text pt={4} fontWeight="bold" color="#55099D">
-            <Link href="/faq">Read our FAQ</Link>
-          </Text>
-        </Box>
-        <StoryForm />
-      </Box>
-    </div>
+    <ContentBox py>
```

Comment:
Again, adopted `ContentBox` for standard page width and responsive padding, also removed the extra nested `Box`.